### PR TITLE
Fixed static css and changed indedx look

### PR DESCRIPTION
**Pull request by SantaPoro**

### CODE REVIEW

File: hackIT/static/styles.css [hunk 1]
```diff
@@ -11,54 +11,25 @@ nav{
...
-	border: 2px solid #02676e;
+	max-width: 900px;
 
-	margin-left:100px;
-	margin-top: 100px;
-
-	padding:10px;
+	padding-right: 10px;
+	padding-left: 10px;
+	background-color: #09CDDA;
```
Comment (Deltaphish) [inline hackIT/static/styles.css]: I think having the text in #09CDDA and the background the same as body would be more subtle for the eyes. Then we get a more "dark mode" experience

Having this #09CDDA for the backround of `Welcome to hackIT` or just the `hackIT` part would be cool tho. And would focus attention to the logo which might be good (honestly no clue).

File: hackIT/templates/index.html.tera [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
     <title>CODE<>IT</title>
   </head>
   <body>
-    <h1>Hello {{name}}</h1>
-    <a href={{auth_url}}>login</a>
-    <h3>{{auth}} </h3>
+    <div class="wrapper">
+      <h1>Welcome to hack_IT!</h1>
+      <p>to acces the challenges, please log in on gamma!</p>
+      <a href="www.google.se"> [=LOGIN=] </a>
```
Comment (Deltaphish) [inline hackIT/templates/index.html.tera]: replace www.google.se with `{{auth}}`. Then gamma will work out of the box

File: hackIT/templates/challenges.html.tera [hunk 3]
```diff
@@ -3,8 +3,10 @@
   <head></head>
```
Comment (Deltaphish) [inline hackIT/templates/challenges.html.tera]: No css for poor `challenges.html.tera`?

**Review by Deltaphish — CHANGES_REQUESTED** [review]
Made some suggestions

File: hackIT/templates/index.html.tera (hunk 2 above)
Comment (SantaPoro) [reply hackIT/templates/index.html.tera]: Did this and it now the link redirects to ./you and says "request not found" or alternatively ./you are not logged in...
not sure of how or why this is happening at the moment

File: hackIT/templates/challenges.html.tera (hunk 3 above)
Comment (SantaPoro) [reply hackIT/templates/challenges.html.tera]: i completely whiffed on this one! it should have some css and with an attempt of making it look nicer (still needs work though)

File: hackIT/static/styles.css (hunk 1 above)
Comment (SantaPoro) [reply hackIT/static/styles.css]: i changed the background and tried what you suggested about the header, i think it looks kind of nice so decided to keep it. The index site as a whole might need a bit more work though but it looks better now!

File: hackIT/templates/index.html.tera (hunk 2 above)
Comment (Deltaphish) [reply hackIT/templates/index.html.tera]: Opps meant auth_url, but we can probably leave it since its outside of the scope of the pr

**Review by Deltaphish — APPROVED** [review]
You are go for merge

**Pull request merged**